test/parallel/test-crypto-sign-verify.js

```diff
@@ -793,3 +793,33 @@ assert.throws(
     }, { code: 'ERR_CRYPTO_UNSUPPORTED_OPERATION', message: 'Unsupported crypto operation' });
   }
 }
+
+{
+  // Dh, x25519 and x448 should not be used for signing/verifying
+  // https://github.com/nodejs/node/issues/53742
+  const keys = [
+    {
+      privateKey: fixtures.readKey('dh_private.pem', 'ascii'),
+      publicKey: fixtures.readKey('dh_public.pem', 'ascii'),
+    },
+    {
+      privateKey: fixtures.readKey('x25519_private.pem', 'ascii'),
+      publicKey: fixtures.readKey('x25519_public.pem', 'ascii'),
+    },
+    {
+      privateKey: fixtures.readKey('x448_private.pem', 'ascii'),
+      publicKey: fixtures.readKey('x448_public.pem', 'ascii'),
+    },
+  ];
+  for (const { publicKey, privateKey } of keys) {
```

Comment:
(Not tested, but shorter.)

```suggestion
  for (const algo of ['dh', 'x25519', 'x448']) {
    const privateKey = fixtures.readKey(`${algo}_private.pem`, 'ascii');
    const publicKey = fixtures.readKey(`${algo}_public.pem`, 'ascii');
```